pandas/core/reshape/encoding.py

```diff
@@ -316,3 +318,202 @@ def get_empty_frame(data) -> DataFrame:
             dummy_mat = dummy_mat[:, 1:]
             dummy_cols = dummy_cols[1:]
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
+
+
+def from_dummies(
+    data: DataFrame,
+    sep: None | str = None,
+    base_category: None | Hashable | dict[str, Hashable] = None,
+) -> DataFrame:
+    """
+    Create a categorical `DataFrame` from a `DataFrame` of dummy variables.
+
+    Inverts the operation performed by :func:`~pandas.get_dummies`.
+
+    Parameters
+    ----------
+    data : DataFrame
+        Data which contains dummy-coded variables.
```

Comment:
Good to specifically mention that `dummy-coded` means integer columns with only 1s and 0s